packages/webapp/src/components/Filter/FilterMultiSelect/index.tsx

```diff
@@ -38,7 +49,7 @@ export const FilterMultiSelect = ({
   }, [shouldReset]);
 
   useEffect(() => {
-    filterRef.current[filterKey] = produce(defaultFilterState, (defaultFilterState) => {
+    filterRef.current![filterKey] = produce(defaultFilterState, (defaultFilterState) => {
```

Comment:
Does it break if you wrap this in `if(filterRef.current) {}`?

This _could_ hide errors otherwise

Reply:
I doubt it would break, but I wanted to avoid changing the JS logic the way a new conditional would (while the non-null assertion is just for the type checker). Since the code works, I'm assuming we don't need the check for functionality? Sounds like @antsgar will be re-writing this one though anyway.

Reply:
Without knowing anything -- even though the code works there might just be an edge case we haven't reached yet but its possible the type checker missed a conditional somewhere else. 🤷

Reply:
The filterRef current value _could_ technically be null so it'd be correct to add the conditional here I think, but this component will all get re-written anyway!